### PR TITLE
Apple login bug fixes

### PR DESCRIPTION
**Pull request by AnsonShaji1**
# Submitting Pull Requests

## General

 - [ ] Make sure you use [semantic commit messages](https://seesparkbox.com/foundry/semantic_commit_messages).
       Examples: `"fix(google): Fixed foobar bug"`, `"feat(accounts): Added foobar feature"`.
 - [ ] All Python code must be 100% pep8 and isort clean.
 - [ ] JavaScript code should adhere to [StandardJS](https://standardjs.com).
 - [ ] If your changes are significant, please update `ChangeLog.rst`.
 - [ ] Feel free to add yourself to `AUTHORS`.
 
 ## Provider Specifics
 
 In case you add a new provider:
 
- [ ] Make sure unit tests are available.
- [ ] Add an entry of your provider in `test_settings.py::INSTALLED_APPS` and `docs/installation.rst::INSTALLED_APPS`.
- [ ] Add documentation to `docs/providers.rst`.
- [ ] Add an entry to the list of supported providers over at `docs/overview.rst`.

### CODE REVIEW

**Comment by bcarradini** [comment]
Does this have any relationship to https://github.com/pennersr/django-allauth/pull/2424?

**Comment by AnsonShaji1** [comment]
Hi,
   We are using allauth version 24.0.1. Because our system has django 1.9
support.  I modify that code.I am trying for implement apple login in that
version.

Regards
Anson

**Comment by bcarradini** [comment]
Oh, wow. Well... good luck!

File: allauth/socialaccount/migrations/0001_initial.py [hunk 1]
```diff
@@ -20,11 +20,11 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('provider', models.CharField(max_length=30, verbose_name='provider', choices=registry.as_choices())),
-                ('uid', models.CharField(max_length=255, verbose_name='uid')),
```
Comment (kelkarajay) [inline allauth/socialaccount/migrations/0001_initial.py:23]: These changes should be introduced in a new migration version

**Review by kelkarajay — CHANGES_REQUESTED** [review]

**Comment by pennersr** [comment]
Apple sign in has just been merged via 16e9c61102de46a52d6c51e7b63fed22865eadb7.

**Pull request closed without merging**